src/requestManager.cpp

```diff
@@ -1196,7 +1196,7 @@ void CRequestManager::FindNextBlocksToDownload(CNode *node, unsigned int count,
 
 void CRequestManager::RequestMempoolSync(CNode *pto)
 {
-    LOCK(cs_mempoolsync);
+    LOCK(cs_mempoolsync_init);
 
```

Comment:
I think you still have to use cs_mempoolsync here because of mempooSyncReqested is guareded under that lock.

Reply:
Ok, I see. Done.